.changeset/olive-lamps-unite.md

```diff
@@ -0,0 +1,5 @@
+---
+"@equinor/mad-components": patch
+---
+
+changed autocomplete to have sub component multiselect
```

Comment:
I believe this should be a minor change, since the API for the `Autoselect` component has breaking changes